Lib/queue.py

```diff
@@ -161,14 +193,25 @@ def get(self, block=True, timeout=None):
         Otherwise ('block' is false), return an item if one is immediately
         available, else raise the Empty exception ('timeout' is ignored
         in that case).
+
+        Raises a ShutDown if shutdown_state attribute is set to
+        `_queue_shutdown` and queue is empty.
+
+        Raises a ShutDown if shutdown_state attribute is set to
+        `_queue_shutdown_immediate`
```

Comment:
```suggestion
        Raises ShutDown if the queue has been shut down and is empty,
        or if the queue has been shut down immediately.
```

Doc-strings are for humans